file.c

```diff
@@ -338,7 +338,6 @@ readrec5(File *f, Job *l, int *err)
             j = make_job_with_id(jr.pri, jr.delay, jr.ttr, jr.body_size,
                                  t, jr.id);
             j->next = j->prev = j;
-            j->r.created_at = jr.created_at;
```

Comment:
Is this line really not necessary?

Reply:
Yep, most of the issues here were reported by cppcheck and manually fixed by me.
In this case, see line 348 where value to `j->r.created_at` assigned regardless of deleted line the branch `if (!j)`.

Reply:
Oh yes, indeed!